.github/pull_request_template.md

```diff
@@ -1,9 +1,10 @@
-<!-- Choose you PR title carefully as it will be used as the entry in the changelog! -->
+<!-- Choose your PR title carefully as it will be used as the entry in the changelog! -->
 
 > [!warning]
-> **This is public repository, ensure not to disclose:**
-> - [ ] personal data beyond what is necessary for interacting with this pull request
-> - [ ] business confidential information, such as customer names
+> **This is a public repository, ensure not to disclose:**
+>
+> - [ ] personal data beyond what is necessary for interacting with this pull request, nor
+> - [ ] business confidential information, such as customer names.
```

Comment:
This is what I was referring to in the other comment, shouldn't it be the same in issues and PRs?